apps/otlp.go

```diff
@@ -49,8 +61,34 @@ func (r ReceiverOTLP) Pipelines() []otel.Pipeline {
 			},
 		},
 		Processors: []otel.Component{
+			// The intent here is to add the workload.googleapis.com prefix to any metrics
+			// that do not match the list of known domains in the exporter [1].
+			//
+			// This would ordinarily be accomplished using a metricstransform processor
+			// with a negative-lookahead regexp, but RE2 (the regexp engine used by
```

Comment:
Nit: I think it's the Go regex package, not RE2, though they support almost the same set of functionality.

Reply:
Updated